src/providers/perplexity.ts

```diff
@@ -7,6 +7,20 @@ const Perplexity: Provider = {
   label: 'Perplexity',
   name: 'perplexity',
   apiKeyUrl: 'https://perplexity.ai/settings/api',
+
+  // Price per 1k tokens [input, output].
+  // Source: https://docs.perplexity.ai/docs/model-cards
+  // Source: https://docs.perplexity.ai/docs/pricing
+  pricing: {
+    'sonar-small-chat': { inputTokensCost: 0.00007, outputTokensCost: 0.00028 },
```

Comment:
[nit] I'm wondering if we should add some more typing for models. Like ProviderModel and then add ModelCost type that would be [key: ProviderModel]: ModelCost or something like that.

Reply:
At the moment we allow for free input of model name, in order to support cases when new models are added since the last release. We only restrict providers. We might consider pros and cons of having a supported model list as a separate issue.